src/vuln_analysis/register.py

```diff
@@ -171,11 +180,13 @@ async def output_results_node(state: AgentMorpheusOutput) -> AgentMorpheusOutput
     subgraph_builder.add_node("agent_executor", agent_executor_node)
     subgraph_builder.add_node("summarize", summarize_node)
     subgraph_builder.add_node("justify", justify_node)
+    subgraph_builder.add_node("generate_cvss", generate_cvss_node)
 
     subgraph_builder.add_edge(START, "checklist")
     subgraph_builder.add_edge("checklist", "agent_executor")
     subgraph_builder.add_edge("agent_executor", "summarize")
     subgraph_builder.add_edge("summarize", "justify")
+    subgraph_builder.add_edge("justify", "generate_cvss")
```

Comment:
@IlonaShishov Why putting the generate_cvss at the end of the pipeline?
in the previous version , it was after `agent_executor` , and just before the `summarize`  node